patches/0062-lavu-add-sub-frame-side-data.patch

```diff
@@ -45,10 +45,18 @@ index 7828c94dc5..1201a08636 100644
  
  OBJS-$(CONFIG_CUDA)                     += hwcontext_cuda.o
 diff --git a/libavutil/frame.c b/libavutil/frame.c
-index b6cee2d886..db423990f0 100644
+index 930f01dc19..40c67de35b 100644
 --- a/libavutil/frame.c
 +++ b/libavutil/frame.c
-@@ -333,7 +333,9 @@ FF_ENABLE_DEPRECATION_WARNINGS
+@@ -56,6 +56,7 @@ static const AVSideDataDescriptor sd_props[] = {
+     [AV_FRAME_DATA_SPHERICAL]                   = { "Spherical Mapping",                            AV_SIDE_DATA_PROP_GLOBAL },
+     [AV_FRAME_DATA_ICC_PROFILE]                 = { "ICC profile",                                  AV_SIDE_DATA_PROP_GLOBAL },
+     [AV_FRAME_DATA_SEI_UNREGISTERED]            = { "H.26[45] User Data Unregistered SEI message",  AV_SIDE_DATA_PROP_MULTI },
++    [AV_FRAME_DATA_SUB_FRAME]                   = { "Sub frame Metadata" },
```

Comment:
Is a AV_SIDE_DATA_PROP_xxx needed ?

Reply:
No, just normal side data type can satisfy our request.